<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimarySkillSignature</class>
 <widget class="QWidget" name="PrimarySkillSignature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>111</width>
    <height>58</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>111</width>
    <height>58</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>111</width>
    <height>58</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelLeft">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>13</y>
     <width>26</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>26</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>26</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRight">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>13</y>
     <width>26</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>26</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>26</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonTop">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>0</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>17</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>17</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonBottom">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>41</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>17</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>17</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCenter">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>17</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelText">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>17</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>NON</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
